appservice/src/createAppService/createWebApp.ts

```diff
@@ -23,28 +23,38 @@ export async function createWebApp(
 }
 
 export async function setWizardContextDefaults(wizardContext: IAppServiceWizardContext, actionContext: IActionContext, advancedCreation?: boolean): Promise<void> {
+    const isJavaProject: boolean = await javaUtils.isJavaProject();
+    if (isJavaProject) {
```

Comment:
@EricJizbaMSFT Do you think we need to enable the multi-root for `js` & `python` here?

The current implementation is separated.

Reply:
I'm not worried about multi-root. According to VS Code folks it has pretty low usage